configs/components/pxp-agent.rb

```diff
@@ -62,9 +62,13 @@
   when "systemd"
     pkg.install_service "ext/systemd/pxp-agent.service", "ext/redhat/pxp-agent.sysconfig"
     pkg.install_configfile "ext/systemd/pxp-agent.logrotate", "/etc/logrotate.d/pxp-agent"
+    if platform.is_deb?
+      pkg.add_postinstall_action ["install"], ["systemctl disable pxp-agent.service >/dev/null || :"]
+    end
   when "sysv"
     if platform.is_deb?
       pkg.install_service "ext/debian/pxp-agent.init", "ext/debian/pxp-agent.default"
+      pkg.add_postinstall_action ["install"], ["update-rc.d pxp-agent disable > /dev/null"]
```

Comment:
I was following what vanagon does with `update-rc.d <%= service.name %> defaults > /dev/null`, but it sounds like it's probably a good idea to add